products/tarantool.md

```diff
@@ -14,7 +14,7 @@ category: db
 iconSlug: NA
 eolColumn: Support Status
 releaseDateColumn: false
-sortReleasesBy: 'releaseCycle'
+sortReleasesBy: 'cycleShortHand'
```

Comment:
We don't have a `cycleShortHand` property defined, so this is same as not sorting at all. It's usually better in such cases to sort by the `release` property, which sorts them by the date of first release instead.

Reply:
@captn3m0 i have fixed that part for @ligurio